packages/angular_devkit/build_angular/src/builders/application/tests/options/polyfills_spec.ts

```diff
@@ -37,7 +37,7 @@ describeBuilder(buildApplication, APPLICATION_BUILDER_INFO, (harness) => {
 
         expect(result?.success).toBe(true);
 
-        harness.expectFile('dist/polyfills.js').toExist();
+        harness.expectFile('dist/bowser/polyfills.js').toExist();
```

Comment:
```suggestion
        harness.expectFile('dist/browser/polyfills.js').toExist();
```
typo.  Same in other parts of file.